aiven/templates/service_user_config_schema.go

```diff
@@ -318,7 +329,7 @@ var (
           "properties": {
             "max_index_count": {
               "example": 3,
-              "maximum": 9223372036854775807,
+              "maximum": 9223372036854776000,
```

Comment:
I wonder why all these maximum values got changed. The API definition is unchanged and `9223372036854775807` is the correct value for all these.

Reply:
Hm... that is strange, they got changed automatically after running `generate_templates.sh`. And indeed after checking  API response `avn service types -v --json | grep 'maximum":'` correct values are 9`223372036854775807`.

Reply:
It looks like `jq` expression from  `generate_templates.sh` script is changing these values